xray/src/generation.rs

```diff
@@ -364,6 +378,38 @@ impl HeightStddevColoringStrategy {
     }
 }
 
+impl ColoringStrategy for HeightStddevColoringStrategy {
```

Comment:
I just moved this one closer to its declaration and added the `attributes()` impl.